src/composer/src/main.rs

```diff
@@ -67,10 +68,26 @@ pub fn exec() -> Result<(), String> {
         .rev()
         .collect();
     for c_id in reverse_ids.iter() {
+        
         sys.add_params_iter(&c_id, Parameters::transition_iter(c_id, &sys, &mut build)?);
+        let obj = ElfObject::transition_iter(c_id, &sys, &mut build)?;
         sys.add_objs_iter(&c_id, ElfObject::transition_iter(c_id, &sys, &mut build)?);
         sys.add_invs_iter(&c_id, Invocations::transition_iter(c_id, &sys, &mut build)?);
+        println!("path:{}", obj.get_path());
+        let output = Command::new("python3")
+        .arg("/home/minghwu/work/minghwu/composite/pyelftool_parser/src/analyzer.py")
```

Comment:
You'll want to use relative paths here so that you can avoid hardcoding the absolute path.

Reply:
Yes, this is indeed a problem. I do not figure out how to put into tools. Therefore, I put absolute path for this moment. I will fix it afterward. Thank you for pointing out.